presto-main/src/main/java/com/facebook/presto/type/DecimalInequalityOperators.java

```diff
@@ -174,6 +199,21 @@ public static boolean opShortShortShortRescale(long a, long b, long aRescale, lo
     }
 
     @UsedByGeneratedCode
+    public static boolean opShortShortShortRescaleNullable(
+            long a, boolean aNull,
+            long b, boolean bNull,
+            long aRescale, long bRescale, MethodHandle getResultMethodHandle)
```

Comment:
I can see how your line break makes sense. But we generally do one parameter per line when we can't fit everything on one line.